packages/playground/src/index.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable no-console */
```

Comment:
No idea why I was suddenly getting these errors, so I added the comments to save time since they blocked rendering of playground.